2019-07-24 12:28:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO MOVIE_RATING(1[*], 1, 10, 10)"; expected "identifier"; SQL statement:
INSERT INTO MOVIE_RATING(1, 1, 10, 10) [42001-192]
2019-07-24 12:57:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1482)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:985)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:959)
	at org.h2.jdbc.JdbcStatement.getResultSet(JdbcStatement.java:205)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1401)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcConnection.getCatalog(JdbcConnection.java:612)
	at org.h2.jdbc.JdbcResultSet.getMetaData(JdbcResultSet.java:143)
	at org.h2.server.web.WebApp.getResultSet(WebApp.java:1627)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1405)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcConnection.getCatalog(JdbcConnection.java:612)
	at org.h2.jdbc.JdbcResultSet.getMetaData(JdbcResultSet.java:143)
	at org.h2.server.web.WebApp.getResultSet(WebApp.java:1627)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1405)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcConnection.getCatalog(JdbcConnection.java:612)
	at org.h2.jdbc.JdbcResultSet.getMetaData(JdbcResultSet.java:143)
	at org.h2.server.web.WebApp.getResultSet(WebApp.java:1627)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1405)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcConnection.getCatalog(JdbcConnection.java:612)
	at org.h2.jdbc.JdbcResultSet.getMetaData(JdbcResultSet.java:143)
	at org.h2.server.web.WebApp.getResultSet(WebApp.java:1627)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1405)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcConnection.getCatalog(JdbcConnection.java:612)
	at org.h2.jdbc.JdbcResultSet.getMetaData(JdbcResultSet.java:143)
	at org.h2.server.web.WebApp.getResultSet(WebApp.java:1627)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1405)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2019-07-24 12:57:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Connection is broken: "session closed" [90067-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.engine.Session.prepareLocal(Session.java:537)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcConnection.getCatalog(JdbcConnection.java:612)
	at org.h2.jdbc.JdbcResultSet.getMetaData(JdbcResultSet.java:143)
	at org.h2.server.web.WebApp.getResultSet(WebApp.java:1627)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1405)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
